src/ci/scripts/install-msys2.sh

```diff
@@ -12,10 +12,12 @@ IFS=$'\n\t'
 source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 
 if isWindows; then
-    for RETRY_COUNT in 1 2 3 4 5 6 7 8 9 10; do
-        choco install msys2 \
-            --params="/InstallDir:$(ciCheckoutPath)/msys2 /NoPath" -y --no-progress \
-            && mkdir -p "$(ciCheckoutPath)/msys2/home/${USERNAME}" \
-            && ciCommandAddPath "$(ciCheckoutPath)/msys2/usr/bin" && break
-    done
+    # Pre-followed the api/v2 URL to the CDN since the API can be a bit flakey
+    curl -sSL https://packages.chocolatey.org/msys2.20190524.0.0.20191030.nupkg > msys2.nupkg
```

Comment:
Have we confirmed that the *API* and not the package registry is flaky? If we're going to pin to a specific version, maybe we should upload these files to S3 and be done with it?

Reply:
> Have we confirmed that the API and not the package endpoint is flaky?

Sort of: https://status.chocolatey.org/issues/2019-12-18-timeouts-when-searching-and-installing-packages/

Reply:
The errors I've seen directly point at the API (https://github.com/rust-lang/rust/issues/67568) and the headers from the CDN indicate it's backed by cloudfront which I figure should be reliable.

This makes it easier to bump versions as well.

Reply:
Okay, great! That sounds like a good reason to prefer the packages endpoint for now, and we can always take the next step if it becomes necessary.